Chapter1/App/authProvider.js

```diff
@@ -0,0 +1,130 @@
+// Create the main MSAL instance
+// configuration parameters are located at authConfig.js
+const pca = new msal.PublicClientApplication(msalConfig);
+
+let username = "";
+let accountId = "";
+
+function selectAccount() {
+
+    /**
+     * See here for more info on account retrieval: 
+     * https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-common/docs/Accounts.md
+     */
+
+    const currentAccounts = pca.getAllAccounts();
```

Comment:
If multiple accounts is not a valid use case, you can also model this as `getAccount()`